### PR TITLE
[Mik317] New disclosures (NPM and Py)

### PR DESCRIPTION
**Pull request by Mik317**
I dunno if this is the correct way to `disclose` the issues, so please be kind if I was wrong lol.
Here I created `7 new disclosures` (6 NPM + 1 Py) which should be reviewed ;)

Regards,
Mik

### CODE REVIEW

**Comment by toufik-airane** [comment]
Can we split the contribution to more atomic reports? :)

**Comment by Mik317** [comment]
Hi @toufik-airane :)
Sure you can :).

I didn't knew how to disclose with this new method, so I've created a `pull request` containing all the disclosures, but now I understood how does it work ;).

Regards,
Mik

**Comment by Mik317** [comment]
Will these bounties be loaded inside the website automatically or I/(someone?) have to revert the changes on my fork and then make single `pull requests` for every bounty?

Regards,
Mik

**Comment by JamieSlome** [comment]
@Mik317 - they will automatically be added to the platform regardless of whether you do many disclosures in a single pull request or one disclosure in a pull request.

Feel free to leave this pull request open and we will deal with the disclosures as soon as possible! 🍰

**Comment by Mik317** [comment]
Hi @JamieSlome :),
thank you for the information then :+1: , I thought you were using some automatic tools that depended on the `build errors` or something similar and it made impossible parse the templates and publish the disclosures.

Thank you a lot :),
Mik

**Comment by JamieSlome** [comment]
@Mik317 - you are correct, we run checks on your pull request to validate that certain information is present in the disclosure.

We will look to support checks on multiple disclosures in the same pull request as well.

File: bounties/npm/count-git-tags/1/README.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+It allows to execute `arbitrary commands remotely inside the victim's PC`
+
+## Module
+* **module name:** `count-git-tags`
+* **version:** `1.0.0`
+* **npm page:** `https://www.npmjs.com/package/count-git-tags`
+
+## Module Description
+> A tool to count the number of tags of a git repository
+
```
Comment (adam-nygate) [inline bounties/npm/count-git-tags/1/README.md:12]: These details are unnecessary for the disclosure

File: bounties/npm/count-git-tags/1/vulnerability.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/aichbauer/node-count-git-tags/",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/count-git-tags/1/vulnerability.json:40]: Replace with `JavaScript`

File: bounties/npm/create-git/1/README.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+It allows to `execute remote commands on the victim's PC`
+
+## Module
+* **module name:** `create-git`
+* **version:** `0.0.4`
+* **npm page:** `https://www.npmjs.com/package/create-git`
+
+## Module Description
+> A generator to initialize a project with git.
+
```
Comment (adam-nygate) [inline bounties/npm/create-git/1/README.md:12]: These details are unnecessary for disclosure

File: bounties/npm/create-git/1/vulnerability.json [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/wesleytodd/create-git",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/create-git/1/vulnerability.json:39]: Replace with `JavaScript`

File: bounties/npm/enpeem/1/README.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+* **module name:** `enpeem`
+* **version:** `1.0.1`
+* **npm page:** `https://www.npmjs.com/package/enpeem`
+
+## Module Description
+> Lightweight wrapper for accessing npm programmatically (alternative to adding npm as a dependency)
+
+## Module Stats
+[1,031] downloads in the last week
+
```
Comment (adam-nygate) [inline bounties/npm/enpeem/1/README.md:15]: Unnecessary for disclosure

File: bounties/npm/enpeem/1/vulnerability.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/balderdashy/enpeem",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/enpeem/1/vulnerability.json:40]: Replace with `JavaScript`

File: bounties/npm/git-commit-range/1/README.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+It allows to execute `arbitrary commands remotely inside the victim's PC`
+
+## Module
+* **module name:** `git-commit-range`
+* **version:** `1.0.1`
+* **npm page:** `https://www.npmjs.com/package/git-commit-range`
+
+## Module Description
+> Get the git commithash within a Range from-to
+
```
Comment (adam-nygate) [inline bounties/npm/git-commit-range/1/README.md:12]: Unnecessary

File: bounties/npm/git-commit-range/1/vulnerability.json [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/aichbauer/node-git-commit-range",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/git-commit-range/1/vulnerability.json:40]: Replace with `JavaScript`

File: bounties/npm/git-commit-range/1/vulnerability.json [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+    ]
+  },
+  "Permalinks": [
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L32",
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L44 ",
+    "https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L46 ",
+    " https://github.com/aichbauer/node-git-commit-range/blob/master/index.js#L50",
```
Comment (adam-nygate) [inline bounties/npm/git-commit-range/1/vulnerability.json:47]: Remove space at the beginning of the string

File: bounties/npm/tagged-git-commits/1/README.md [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+It allows to execute `arbitrary commands remotely inside the victim's PC`
+
+## Module
+* **module name:** `tagged-git-commits`
+* **version:** `1.0.0`
+* **npm page:** `https://www.npmjs.com/package/tagged-git-commits`
+
+## Module Description
+> Get the commit hash and refs/tags of tagged commits, remote and local
+
```
Comment (adam-nygate) [inline bounties/npm/tagged-git-commits/1/README.md:12]: Unnecessary

File: bounties/npm/tagged-git-commits/1/vulnerability.json [hunk 11]
```diff
@@ -0,0 +1,54 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/aichbauer/node-tagged-git-commits",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/tagged-git-commits/1/vulnerability.json:40]: Replace with `JavaScript`

File: bounties/npm/tagged-git-commits/1/vulnerability.json [hunk 12]
```diff
@@ -0,0 +1,54 @@
...
+  },
+  "Permalinks": [
+    "https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L29",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L30",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L32",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L33",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L39",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L40",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L42",
+    " https://github.com/aichbauer/node-tagged-git-commits/blob/master/index.js#L43"
```
Comment (adam-nygate) [inline bounties/npm/tagged-git-commits/1/vulnerability.json:51]: Remove space at the beginning of the string

File: bounties/npm/wdio-testrecorder-reporter/1/vulnerability.json [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+    "E": "",
+    "RL": "",
+    "RC": "",
+    "Score": "8.3"
+  },
+  "CVEs": [],
+  "Repository": {
+    "URL": "https://github.com/tamil777selvan/wdio-testrecorder-reporter",
+    "Codebase": [
+      "NodeJS"
```
Comment (adam-nygate) [inline bounties/npm/wdio-testrecorder-reporter/1/vulnerability.json:40]: Replace with `JavaScript`

File: bounties/npm/wdio-testrecorder-reporter/1/README.md [hunk 14]
```diff
@@ -0,0 +1,35 @@
...
+* **module name:** `wdio-testrecorder-reporter`
+* **version:** `1.0.2`
+* **npm page:** `https://www.npmjs.com/package/wdio-testrecorder-reporter`
+
+## Module Description
+> A reporter for WDIO v5 that generates videos of your wdio test executions.
+
+## Module Stats
+[166] downloads in the last week
+
```
Comment (adam-nygate) [inline bounties/npm/wdio-testrecorder-reporter/1/README.md:15]: Unnecessary

File: bounties/python/fsociety/1/vulnerability.json [hunk 15]
```diff
@@ -0,0 +1,52 @@
...
+      "Python"
+    ]
+  },
+  "Permalinks": [
+    "https://github.com/Manisso/fsociety/blob/master/fsociety.py#L485",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L488",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L491",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L583",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L587",
+    " https://github.com/Manisso/fsociety/blob/master/fsociety.py#L636"
```
Comment (adam-nygate) [inline bounties/python/fsociety/1/vulnerability.json:49]: Remove spaces at the beginning of the strings

File: bounties/npm/count-git-tags/1/README.md (hunk 1 above)
Comment (Mik317) [reply bounties/npm/count-git-tags/1/README.md]: Removed in my `master` branch

File: bounties/npm/count-git-tags/1/vulnerability.json (hunk 2 above)
Comment (Mik317) [reply bounties/npm/count-git-tags/1/vulnerability.json]: Replaced in my `master` branch

File: bounties/npm/create-git/1/README.md (hunk 3 above)
Comment (Mik317) [reply bounties/npm/create-git/1/README.md]: Removed them :)

File: bounties/npm/create-git/1/vulnerability.json (hunk 4 above)
Comment (Mik317) [reply bounties/npm/create-git/1/vulnerability.json]: Done

File: bounties/npm/enpeem/1/README.md (hunk 5 above)
Comment (Mik317) [reply bounties/npm/enpeem/1/README.md]: Removed 'em

File: bounties/npm/enpeem/1/vulnerability.json (hunk 6 above)
Comment (Mik317) [reply bounties/npm/enpeem/1/vulnerability.json]: Done

File: bounties/npm/git-commit-range/1/README.md (hunk 7 above)
Comment (Mik317) [reply bounties/npm/git-commit-range/1/README.md]: Removed

File: bounties/npm/git-commit-range/1/vulnerability.json (hunk 8 above)
Comment (Mik317) [reply bounties/npm/git-commit-range/1/vulnerability.json]: - [ x ]

File: bounties/npm/git-commit-range/1/vulnerability.json (hunk 9 above)
Comment (Mik317) [reply bounties/npm/git-commit-range/1/vulnerability.json]: Done

File: bounties/npm/tagged-git-commits/1/README.md (hunk 10 above)
Comment (Mik317) [reply bounties/npm/tagged-git-commits/1/README.md]: Removed

File: bounties/npm/tagged-git-commits/1/vulnerability.json (hunk 11 above)
Comment (Mik317) [reply bounties/npm/tagged-git-commits/1/vulnerability.json]: Done

File: bounties/npm/tagged-git-commits/1/vulnerability.json (hunk 12 above)
Comment (Mik317) [reply bounties/npm/tagged-git-commits/1/vulnerability.json]: Done

File: bounties/npm/wdio-testrecorder-reporter/1/README.md (hunk 14 above)
Comment (Mik317) [reply bounties/npm/wdio-testrecorder-reporter/1/README.md]: Removed

File: bounties/npm/wdio-testrecorder-reporter/1/vulnerability.json (hunk 13 above)
Comment (Mik317) [reply bounties/npm/wdio-testrecorder-reporter/1/vulnerability.json]: Done

File: bounties/python/fsociety/1/vulnerability.json (hunk 15 above)
Comment (Mik317) [reply bounties/python/fsociety/1/vulnerability.json]: Done

**Comment by Mik317** [comment]
Hi @adam-nygate :)
Thanks a lot for your help ;)
I hope I've resolved all the issues as well :)

Regards,
Mik

**Pull request merged**